projects/canopy/src/lib/tabs/tabs.component.html

```diff
@@ -1,8 +1,10 @@
-<div class="lg-tabs__list" role="tablist" [attr.aria-label]="label">
+<div class="lg-tabs__list">
   <lg-heading
     class="lg-tabs__list-item-heading"
     *ngFor="let tab of tabs; let index = index"
     [level]="headingLevel"
+    role="tablist"
```

Comment:
This won't address the issue. What you've done here is make each tab an element with the `tablist` role, so we end up with multiple tablists, each with a single tab instead of one tablist with multiple tabs.

The issue in #676 , is that the button (from line 9) has the `role="tab"` attribute rather than the grandparent, `lg-heading`. For a tablist to be accessible, the elements with the `role="tab"` must be a direct child of the tablist element (or have their id as part of the `aria-owns` property of a tablist).

I think the tablist element needs to remain as it was, and either the direct children have the `role="tab"` attribute or possibly the order of the elements needs to be changed (so the `lg-heading` is a child of the `button` element).

Reply:
I committed the suggested changes. Thank you.